lib/js/components/ProgressBar/ProgressBar.vue

```diff
@@ -226,6 +231,15 @@ $progress-bar-layers: (
 		position: absolute;
 		top: 0;
 	}
+
+	&__resultsShadow {
```

Comment:
told ya, "use `background` for ranges, then eveything works". Includign shadows :trollface: